RFC-676-Contributor-Council-Charter.md

```diff
@@ -0,0 +1,221 @@
+recorded on the change proposal. Opinion submissions follow the [Chatham House Rule](https://www.ibabs.com/en/glossary/chatham-house-rule/#:~:text=The%20Chatham%20House%20Rule%20reads,')
+by default, though Council members may choose to sign them.
+  * If the change is deemed appropriate for inclusion in CDK by the Council, the AWS CDK Product team decides whether the feature will be built by
+AWS and adds it to their feature backlog, or if the feature will be offered to the community for implementation.
+  * If a change is deemed to be appropriate for community-led development, the label 'open-for-community-contribution' will be added to the feature
+request.
+  * It is preferred that community-led features are code-reviewed by a
+[Distinguished Contributor](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#badges-pilot-program) before AWS reviews them.
+    * If a Distinguished Contributor is not available within 30 days, AWS will review.
+    * The feature needs to adhere to all AWS development and security processes, which AWS will publish on the CDK GitHub. AWS is committed to
+providing all the required resources for timely reviews and merging.
+    * Those who build a feature through to completion are recognized in a manner to be determined.
+
+## Structure of the Council
+
+The Council is led by two Chairpersons. They do not submit opinions in the decision-making process:
+
+* One Co-Chair and the facilitator of Council sessions will come from the AWS Open Source Strategy & Marketing (OSSM) team.
+* The other Co-Chair is the AWS CDK Product Lead. They set the priorities for AWS resources, but do not have veto power over what the community can
+or can’t work on.
+
+Amazon will have five permanent members on the Council, which AWS may rotate as necessary. They will represent various AWS teams with vested
+interests in CDK to bring a multiplicity of viewpoints and experiences to Council debate (Engineering, Support, OSSM).
+
+The community will have five elected members on the Council.
+
+* The community representatives will be nominated and chosen by the CDK community through a public, time-constrained mechanism which permits
+transparency into the mechanism and allows the community to self-identify—and make an informed decision about—people who are actively looking to
+make deeper investment into CDK.
+* In order to participate in the full-breadth of conversation related to CDK and the project’s roadmap, Council members will be required to sign an
+NDA.
+* The five elected community representatives serve one-year terms. There is no limit to the number of subsequent terms a Council member may serve.
+Depending on community input, the AWS CDK team may decide to allocate a certain number of Council seats to specific community groups, such as the
+Open Construct Foundation, or Enterprise Customers. The rest of the seats would be undesignated.
```

Comment:
A little confused on the wording here. Are these special interest group seats in addition to the five elected members?